test/parallel/test-fs-mkdtemp-prefix-check.js

```diff
@@ -3,7 +3,7 @@ const common = require('../common');
 const assert = require('assert');
 const fs = require('fs');
 
-const prefixValues = [undefined, null, 0, true, false, 1, ''];
+const prefixValues = [undefined, null, 0, true, false, 1];
```

Comment:
behavioral change? I think it was intentional that empty strings were disallowed.

Reply:
@addaleax Did you see the discussion under https://github.com/nodejs/node/pull/39028#discussion_r650593401?

Reply:
It’s fine if this is an intentional change here, but then let’s maybe add a `changes:` entry since this an API change

Reply:
Agreed. I think, we could also turn `prefix` into an optional parameter in a follow-up PR.

Reply:
Let’s **not** do that – People really should provide a prefix unless they have a good reason not to specify one (which I can’t think of, which is why I commented here).